content/english/python-tensorflow/activity-2.md

```diff
@@ -22,26 +24,42 @@ plt.show() #displays the entire plot
 ```
 
 {{% notice tip %}}
-### Question 1
 
-Change the index of the “train_images[x]” and look at the color bar on the right, what is the max pixel value?
+Notice that the image generated has a color bar on the left. This metric is showing that the shoe uses pixel values from 0 to 255. What are pixel values? 
+
+Pixels are the smallest unit of a graphic (image or display) that a screen can represent. Think of your computer screen! There are millions of tiny colored dots that when combined can create a picture of a boat or the text shown on your screen. 
+
+The color bar shows the range in the color intensity that a pixel can display. 
 ![A2 Q1](../media/a2q1.png)
 {{% /notice %}}
 
-{{% notice tip %}}
-### Question 2
+#### Question 1:
+After analyzing the shoe, we want to view another item from the clothing sample. Can you figure out how to do that using the previous code block? 
 
-After analyzing the shoe, you now take another clothing item from your sample and do the same thing. Change the index value to whatever you like so you have a different item. 
-Save this value for later as you will use this in Activity 6. 
+{{% notice hint %}}
+Use the code in the previous code block. Then change the index value from of the training images (“train_images[x]”) to get a new image of an item.   
 
 For example, if you input the index value as 7, you get the following item:
 ![Shirt](../media/a2progress2.png)
 
 {{% /notice %}}
 
-## Copy the following code onto Google Colab:
+### Normalize Pixel Values:
+
+The program we're going to write takes an input of values between 0 and 1. However, our pixel values are mostly all greater than 1! In fact, the range of values is 0 through 255. How can we change the range of numbers such that it can be inputted into our program? 
+
+We will use a process called "normalization," where we transform these values to make them fit in the range of 0 to 1. More specifically, we will take all of our data and divide it by a singular value so that the range of values can now fit within 0 and 1. 
```

Comment:
```suggestion
We will use a process called **"normalization"**, where we transform these values to make them fit in the range of 0 to 1. More specifically, we will take all of our data and divide it by a singular value so that the range of values can now fit within 0 and 1. 
```